src/components/QuranReader/EndOfScrollingControls/index.tsx

```diff
@@ -24,21 +28,30 @@ const EndOfScrollingControls: React.FC<Props> = ({
   initialData,
 }) => {
   return (
-    <div className={styles.container}>
-      <div className={styles.buttonsContainer}>
-        {quranReaderDataType === QuranReaderDataType.Chapter && (
-          <ChapterControls lastVerse={lastVerse} initialData={initialData} />
-        )}
-        {(quranReaderDataType === QuranReaderDataType.Verse ||
-          quranReaderDataType === QuranReaderDataType.VerseRange) && (
-          <VerseControls lastVerse={lastVerse} />
-        )}
-        {quranReaderDataType === QuranReaderDataType.Page && <PageControls lastVerse={lastVerse} />}
-        {quranReaderDataType === QuranReaderDataType.Juz && <JuzControls lastVerse={lastVerse} />}
-        {quranReaderDataType === QuranReaderDataType.Rub && <RubControls lastVerse={lastVerse} />}
-        {quranReaderDataType === QuranReaderDataType.Hizb && <HizbControls lastVerse={lastVerse} />}
+    <>
+      <RevelationOrderNavigationNotice
```

Comment:
We need to conditionally render this because now it shows all the time regardless whether the revelation mode is on/off.

Reply:
Also I believe we should only show it when `quranReaderDataType === QuranReaderDataType.Chapter` because it's not relevant to a Juz/page/hizb navigation